test-adapter/testadapter/tinktestrunner/Injector.hx

```diff
@@ -5,6 +5,24 @@ import haxe.macro.Context;
 import haxe.macro.Expr;
 
 class Injector {
+	public static function buildCase():Array<Field> {
+		var fields = Context.getBuildFields();
+		for (field in fields) {
+			switch (field.name) {
+				case "shouldRun":
+					field.name = "__" + field.name;
+				case _:
+			}
+		}
+
+		var extraFields = (macro class {
+			public function shouldRun(includeMode:Bool):Bool {
+				return testadapter.data.TestFilter.shouldRunTest($v{Macro.filters}, clazz, caze.info.name) && __shouldRun(includeMode);
```

Comment:
Err, where are `clazz` and `caze.info.name` meant to come from here? Have even tried to compile this? :P

Reply:
I guess caze should probably be `this`, but cases don't have a way to get their classnames... and in our case we need a full name with package and classname